tests/testapps/CTCatalog/CTCatalog/Tests/CTCLineTestViewController.mm

```diff
@@ -35,7 +35,7 @@ - (void)drawRect:(CGRect)rect {
 
     // Flips y-axis for our frame
     CGContextScaleCTM(context, 1.0f, -1.0f);
-    CGContextSetTextPosition(context, 0.0, 10.0);
+    CGContextSetTextPosition(context, 0.0, 0.0);
```

Comment:
Changed this when I was trying to find out where the problem was coming from.  Works either way, and iirc Nithish wanted me to change the hardcoded 10 (though 0 isn't much better)